### PR TITLE
Updated link to App Center sign up page

### PR DESCRIPTION
**Pull request by elamalani**
Updated link to App Center sign up page.

### CODE REVIEW

File: labs/vstsextend/appcenter/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ folder: /labs/vstsextend/appcenter/
 
 ## Overview
 
-[Visual Studio App Center](https://appcenter.ms/) is an integrated mobile development lifecycle solution for iOS, Android, Windows and macOS apps. It brings together multiple services commonly used by mobile developers, including build, test, distribute, monitor, diagnose , etc., into one single integrated cloud solution.
+[Visual Studio App Center](https://appcenter.ms/signup?utm_source=Azure%20DevOps%20Lab&utm_medium=Azure) is an integrated mobile development lifecycle solution for iOS, Android, Windows and macOS apps. It brings together multiple services commonly used by mobile developers, including build, test, distribute, monitor, diagnose , etc., into one single integrated cloud solution.
```
Comment (jwallra) [inline labs/vstsextend/appcenter/README.md]:
```suggestion
[Visual Studio App Center](https://appcenter.ms/signup?utm_source=Azure%20DevOps%20Lab&utm_medium=Azure) is an integrated mobile development lifecycle solution for iOS, Android, Windows and macOS apps. It brings together multiple services commonly used by mobile developers, including build, test, distribute, monitoring, diagnostics , etc., into one single integrated cloud solution.
```

**Comment by elamalani** [comment]
#sign-off

File: labs/vstsextend/appcenter/README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ folder: /labs/vstsextend/appcenter/
 
 ## Overview
 
-[Visual Studio App Center](https://appcenter.ms/) is an integrated mobile development lifecycle solution for iOS, Android, Windows and macOS apps. It brings together multiple services commonly used by mobile developers, including build, test, distribute, monitor, diagnose , etc., into one single integrated cloud solution.
+[Visual Studio App Center](https://appcenter.ms/signup?utm_source=Azure%20DevOps%20Lab&utm_medium=Azure) is an integrated mobile development lifecycle solution for iOS, Android, Windows and macOS apps. It brings together multiple services commonly used by mobile developers, including build, test, distribute, monitoring, diagnostics , etc., into one single integrated cloud solution.
```
Comment (dhei) [inline labs/vstsextend/appcenter/README.md]:
```suggestion
[Visual Studio App Center](https://appcenter.ms/signup?utm_source=Azure%20DevOps%20Lab&utm_medium=Azure) is an integrated mobile development lifecycle solution for iOS, Android, Windows and macOS apps. It brings together multiple services commonly used by mobile developers, including build, test, distribute, monitoring, diagnostics, etc., into one single integrated cloud solution.
```

**Comment by elamalani** [comment]
@dmckinstry @sriramdasbalaji can you please review and merge it?

Thanks!

**Comment by dmckinstry** [comment]
@elamalani - We'll push changes through the pipeline along with approvals within the next two days.  It's all automated but also gated and I want to review these along with a couple of others with the team before going live.  Thank you for contributing!

**Comment by elamalani** [comment]
@dmckinstry Looks like the PR is merged but I don't see the changes live. When is it supposed to be live? Please let me know.

**Pull request merged**